charon/src/lib.rs

```diff
@@ -24,6 +24,9 @@
 #![feature(let_chains)]
 #![feature(lint_reasons)]
 #![feature(trait_alias)]
+#![feature(register_tool)]
+// For when we use charon on itself :3
```

Comment:
I prefer avoiding smileys in the code... But smileys in the github conversations are of course ok. ;)